### ISSUE TITLE
Using buildWebhookTopic

### ISSUE THREAD

**Issue by IvanovPavel** [romanzipp/Laravel-Twitch#66]
When I use **buildWebhookTopic** in new lib version I've got an error **400 Bed Request**.
I've spent a some debug execution and reason that you have double json_encode
One of them in
```
 if (null !== $body) {
            $jsonBody = json_encode($body);
        }
```
and another then you call a Guzzle request, cause '**json**' param in `Guzzle` request must be an array. So `Guzzle` by himself encode data.

**Comment by IvanovPavel** [comment]
to fix it you just remove `json_encode` and it will be work
```
        if (null !== $body) {
            $jsonBody = $body;
        }
```

**Comment by romanzipp** [comment]
Thanks for the report. I've released version [3.0.4](https://github.com/romanzipp/Laravel-Twitch/releases/tag/3.0.4) regarding this issue.
